sys-devel/gcc_bootstrap/gcc_bootstrap-11.2.0_2021_07_28.recipe

```diff
@@ -183,7 +183,7 @@ BUILD()
 		--target=$effectiveTargetMachineTriple \
 		--prefix=$installDir --libexecdir=$installDir/lib --mandir=$manDir \
 		--docdir=$docDir --enable-threads=posix \
-		--disable-nls --enable-shared --with-gnu-ld --with-gnu-as \
+		--disable-nls --enable-shared --disable-plugin --with-gnu-ld --with-gnu-as \
```

Comment:
weird.. seems fine though.  I'm assuming the --disable-plugin won't impact ports that use an unbootstrapped gcc?

Reply:
It really just looks like a long command line containing many files with long path names while running "make install-plugin" and in my build environment it somehow reached a limit.

According to gcc docs the "plugins" are used only for code instrumentation and static analysis so probably we can live without them during the bootstrap phase.